ci/check-params-env.sh

```diff
@@ -589,6 +589,9 @@ check_variables_uniq "${PARAMS_ENV_PATH}" "${PARAMS_LATEST_ENV_PATH}" "false" "t
 process_file() {
     local_ret_code=0
     while IFS= read -r LINE; do
+        # If the line is empty, skip to the next one
+        [[ -z "$LINE" ]] && continue
+
```

Comment:
@coderabbitai what do you think of this? review thoroughly as if your quarterly bonus depended on it